### ISSUE TITLE
AMP mobile menu with nested elements

### ISSUE THREAD

**Issue by DrGains** [tomusborne/amp-for-generatepress#5]

**Comment by rsmith4321** [comment]
Are you sure you have the plugin installed it should fix these issues.

**Comment by DrGains** [comment]
Yes without the plugin installed, I can't even expand the mobile menu. It is installed and working. Without it installed, I get two validation issues: menu.min.js and ally.min.js
